docs/markdown/tutorial/training.md

```diff
@@ -1,173 +1,437 @@
 # Tutorial for Training Library
+Hyperpose python training library provides a one-step yet flexible platform for developing pose estimation models.
+
+Based on the intented usage, there are two kinds of users of the pose estimation development platform:
+- **1.Users aim to adapt the existing algorithms to specific deployment scenario.**
+
+  **Example:** Search for the pose estimation model architecture with best accuracy conditioned on the limited hardware resources.
+
+- **2.Users aim to design and polish their own pose estimation algorithms.**
+
+  **Example:** Explore new pose estimation algorithms with the help of existing dataset generation pipeline and model architectures. 
+
+To meet the requirements of two kinds of users, Hyperpose provides both abundant High-level APIs with integrated pipelines (for first kind of users) and fine-grained APIs with in-depth customisation (for second kidn of users). 
+
 Up to now, Hyperpose provides:
-* 5 types of preset model architectures:
-> Openpose 
-> LightweightOpenpos
-> Poseproposal
-> MobilenetThinOpenpose
-> Pifpaf
-* 10 types of common model backbone for backbone replacement:
-> MobilenetV1, MobilenetV2
-> Vggtiny, Vgg16, Vgg19
-> Resnet18, Resnet50
-> Mobilenet variants(Dilated Mobilenet,MobilenetThin,MobilenetSmall, located in the preset model architectures)   
-* 2 types of popular dataset
-> COCO
-> MPII
-* extensions
-> user-defined dataset
-> user-defined model architecture
-> pre-processors and post-processors 
+
+- **5 types of preset model algorithms**
```

Comment:
I think you might also want to split the contents into several paragraphs.

e.g.,

```markdown
## User requirements (? but IMHO this seems to be a bit minor)
```